b2sdk/v2/transfer.py

```diff
@@ -7,29 +7,48 @@
 # License https://www.backblaze.com/using_b2_code.html
 #
 ######################################################################
+from concurrent.futures import Future, ThreadPoolExecutor
+from threading import Lock
+from typing import Callable
 
 from b2sdk import _v3 as v3
 
 
-class LazyThreadPool(v3.LazyThreadPool):
+class LazyThreadPool:
```

Comment:
it looks like it shares the interface with `ThreadPoolMixin`, but not explicitly?

Reply:
After [fc9de68](https://github.com/Backblaze/b2-sdk-python/pull/297/commits/fc9de68873550ac431cee0d257bb714d1ebee152) it shares the `submit` method. Is that a problem?

Reply:
Not anymore.